gatekeeper/src/main/java/eu/arrowhead/core/gatekeeper/GatekeeperApplicationInitListener.java

```diff
@@ -63,6 +65,9 @@
 	@Autowired
 	private SSLProperties sslProps;
 	
+	@Autowired
+	private RelaySupervisingTask relayQueueRemovalThread;
```

Comment:
is this used here? I don't find it.

if it used, please rename the variable to match the class name.

Reply:
not used